src/main/java/com/zufar/onlinestore/payment/service/PaymentMethodGetter.java

```diff
@@ -0,0 +1,40 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.PaymentMethod;
+import com.stripe.param.PaymentMethodCreateParams;
+import com.zufar.onlinestore.payment.config.StripeConfiguration;
+import com.zufar.onlinestore.payment.dto.CreatePaymentMethodDto;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentMethodGetter {
```

Comment:
it seems that this class is creator, not getter

Reply:
Done